version.pri

```diff
@@ -1,4 +1,4 @@
-PROJECT_NAME = qbittorrent
+PROJECT_NAME = qBittorrent
```

Comment:
I don't think this is wise. It changes the name of the produced tarball. I think there is an unwritten rule to have tarballs in lowercase.
And probably distros have scripts that check for new tarballs which will break. eg [debian/watch](https://anonscm.debian.org/cgit/collab-maint/qbittorrent.git/tree/debian/watch)